docs/.vuepress/components/cdr-doc-code-snippet.vue

```diff
@@ -168,14 +168,14 @@ export default {
     border-bottom: 0;
     border-radius: $cdr-doc-border-radius-default $cdr-doc-border-radius-default 0 0;
     display: flex;
-    padding: $inset-half-x $inset-1-x;
+    padding: 10px $inset-1-x;
```

Comment:
again did Corey get this validated by design?

Reply:
Doubtful. The entire doc site is an example of skirting the design system.

"3761 - Can we increase padding top/bottom to 10px? Sorry, Icons are a bit too large and it may be easier to change the padding." -- Corey

Reply:
@bboehlke can you validate this is what design really wants to do please?

Reply:
Let's stick with the system and update the 10px to $space-three-quarter-x (12px)